gubernator/gcs_async.py

```diff
@@ -26,7 +27,17 @@
 @ndb.tasklet
 def get(url):
     context = ndb.get_context()
-    headers = {'accept-encoding': 'gzip, *', 'x-goog-api-version': '2'}
+
+    headers = {
```

Comment:
This code should only execute if the URL is a cloud endpoint. You can use urlparse.urlparse to check the domain against a whitelist.

Reply:
hummm, but why the test failed?